helm/hpcc/templates/eclccserver.yaml

```diff
@@ -54,9 +54,9 @@ data:
       template:
         metadata:
           labels:
+            {{- include "hpcc.addStandardLabels" (dict "root" $ "component" .me.name "name" "eclccserver" "instance" $compileJobName ) | indent 12 }}
```

Comment:
> "component" .me.name

I think this should be "eclccserver" (NB: it's really what would be 'eclcc' in bare-metal - but it is rolled into 1 in k8s)
.me.name is going to become whatever name the eclccserver in this list has been given (which happens to be 'eclccserver' in the example values.yaml)